### ISSUE TITLE
[fr] false alarms PAS_DE_VIRGULE

### ISSUE THREAD

**Issue by jaumeortola** [languagetool-org/languagetool#4576]
> Si le phénomène existe toujours, il se double aujourd’hui d’une autre dimension, dans l’espace virtuel du Web et des réseaux sociaux**, et** délaisse les questions spirituelles au profit de la santé et du développement personnel.

This seems a false alarm.

**Comment by udomai** [comment]
Good point!

@Rebecca-auque, what do you say, a possible solution could be an AP like this:

```xml
<token negate_pos="yes" postag="V.*[123].*" postag_regexp="yes"/>
<token>,</token>
<token>et|&amp;|ou</token>
<token postag="V.*[123].*" postag_regexp="yes"/>

**Comment by Rebecca-auque** [comment]
Indeed, there seemed to be a problem with one of the APs. The problem should be fixed. I also extended the rule adding the &amp; to the rule.
